src/index.ts

```diff
@@ -29,26 +29,26 @@ export const generateKey = (mnemonic?: string) => {
   return getKeyPairFromPath(mnemonic, path);
 };
 
-export const exportPrivateKey = (key: ec.KeyPair) =>
+export const exportPrivateKey = (key: ec.KeyPair): string =>
```

Comment:
By convention in TS we don't put the return type if it's inferred by the compiler which seems to be the case everywhere in this file

Reply:
Ah yes, not sure why I added those.